src/code42cli/output_formats.py

```diff
@@ -76,6 +77,30 @@ def _requires_list_output(self):
         return self.output_format in (OutputFormat.TABLE, OutputFormat.CSV)
 
 
+class DataFrameOutputFormatter:
+    def __init__(self, output_format):
```

Comment:
Wow! Thanks for doing this

Reply:
I am currently facing a bug with CSV windows output formatter and now I am wondering if this fixes it...https://stackoverflow.com/questions/3348460/csv-file-written-with-python-has-blank-lines-between-each-row